### ISSUE TITLE
[a11y] Cannot type in Agent search form

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1421]
**Describe the bug**
I cannot enter text in the Last Name or First Name fields, which are both visible and appear to be editable.

**To Reproduce**
1. Go to New Exchange In
2. Try to search Agents from Recieved From
3. See error

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/167204421-e1b8138e-960d-4857-9985-0aa62f593ccb.png)

`edge` has no issue
![image](https://user-images.githubusercontent.com/37256050/167204457-ee7171f0-1ed6-468c-8bfb-ff9d169b1237.png)

**Desktop (please complete the following information):**
macOS, Edge
